src/Traits/CsvDownloadTrait.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PNX\DrupalTestUtils\Traits;
+
+use League\Csv\CharsetConverter;
+use League\Csv\Reader;
+
+/**
+ * CSV download trait.
+ */
+trait CsvDownloadTrait {
+
+  /**
+   * Asserts that a CSV export has a certain number of rows.
+   *
+   * Clicks the orange 'CSV' button displayed in views, waits for batch to run,
+   * then asserts contents of download.
+   */
+  protected function assertViewsCsvExportRowCount(int $count, bool $isBatch = TRUE): Reader {
+    $this->clickLink('Download CSV');
```

Comment:
I think we should make this a default parameter in case sites are overriding the link text

Reply:
Good idea